samples/Commerce/Commerce.UI/Views/ProductControl.xaml

```diff
@@ -90,32 +90,32 @@
 								<Button Content="add to cart"
 										x:Uid="_AddToCart"
 										utu:AutoLayout.CounterAlignment="Stretch"
-										Style="{StaticResource MaterialContainedButtonStyle}" />
+										Style="{StaticResource ContainedButtonStyle}" />
```

Comment:
Contained is now either Elevated, Filled, or Filled Tonal

https://m3.material.io/components/buttons/overview